rclcpp/test/rclcpp/test_time.cpp

```diff
@@ -86,44 +86,124 @@ TEST(TestTime, time_sources) {
   EXPECT_NE(0u, steady_now.nanosec);
 }
 
-TEST(TestTime, conversions) {
+static const int64_t HALF_SEC_IN_NS = 500 * 1000 * 1000;
+static const int64_t ONE_SEC_IN_NS = 1000 * 1000 * 1000;
```

Comment:
2 * HALF_SEC_IN_NS?

Reply:
I do not see how that would improve readability. But I can change to
```cpp
static const int64_t HALF_SEC_IN_NS = RCL_MS_TO_NS(500);
static const int64_t ONE_SEC_IN_NS = RCL_S_TO_NS(1);
static const int64_t ONE_AND_HALF_SEC_IN_NS = RCL_MS_TO_NS(1500);
```
?

The code has originally been copied from
https://github.com/ros2/rclcpp/blob/a667583821fb3b1c0830420ce3b0517df1e9f8c4/rclcpp/test/rclcpp/test_duration.cpp#L140-L141
and I added `ONE_SEC_IN_NS` to cover the case where the `nanosec` part is zero during message conversion.